devel/mps_load_test.sh

```diff
@@ -215,11 +215,11 @@ make -j8 CONF=$compile_mode apache_trace_stress_test_server \
 
 # If a custom .so got specified, install it.
 if [[ -n "$custom_so" ]]; then
-  install -c $custom_so /usr/local/apache2/modules/mod_pagespeed.so
+  install -c $custom_so $HOME/apache2/modules/mod_pagespeed.so
```

Comment:
Sorry -- we already have the variable I was talking about.  Use $APACHE_DEBUG_ROOT here, defined above as I described.